public/app/routes/routes.ts

```diff
@@ -34,7 +34,7 @@ export function setupAngularRoutes($routeProvider: route.IRouteProvider, $locati
         component: importDashboardPage,
       },
     })
-    .when('/d/:uid/:slug', {
+    .when('/d/:uid', {
```

Comment:
This is the wrong route to update. Need to copy line 73 to 84 and paste as new line after line 73 where the route should be updated to `/d-solo/:uid`. Makes sense?

Reply:
Oh I got it. I have updated the PR.